src/coreclr/jit/codegencommon.cpp

```diff
@@ -2315,9 +2315,9 @@ void CodeGen::genReportEH()
 
         CORINFO_EH_CLAUSE_FLAGS flags = ToCORINFO_EH_CLAUSE_FLAGS(HBtab->ebdHandlerType);
 
-        if (isNativeAOT && (XTnum > 0))
+        if (XTnum > 0)
         {
-            // For NativeAOT, CORINFO_EH_CLAUSE_SAMETRY flag means that the current clause covers same
+            // For NativeAOT and the new coreclr EH, CORINFO_EH_CLAUSE_SAMETRY flag means that the current clause covers same
```

Comment:
It may be a good idea to submit the JIT change as separate PR to make it easier to review and verify that it does not regress the existing EH.